### PR TITLE
Feat: Create Type for Journal in Typescript

### PR DESCRIPTION
**Pull request by ydymegan**
# Notion Task
https://www.notion.so/70204328dfbc4abfa76a39d9c3b50169v=21f1019a9ecd49b791569d2ce69e5cc7&p=a087a611589a419e9f15a07a17af662c

# Changes Made In This PR
- created journal.ts file under `src/types`
- created 4 new types (JournalEntry, WPLHDashboard, Piechart and Scurve)

### CODE REVIEW

File: src/types/Journal.ts [hunk 1]
```diff
@@ -0,0 +1,41 @@
+export type JournalEntry = {
+  id: string;
+  uid: string;
+  entryDate: number;
```
Comment (millasml) [inline src/types/Journal.ts:4]: call this "createdAt"

File: src/types/Journal.ts [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  lastEditedDate: number;
```
Comment (millasml) [inline src/types/Journal.ts:5]: call this "updatedAt" 

(following what Resume.ts does)

File: src/types/Journal.ts [hunk 3]
```diff
@@ -0,0 +1,41 @@
+export type JournalEntry = {
+  id: string;
+  uid: string;
```
Comment (millasml) [inline src/types/Journal.ts:3]: call this "user"

Comment (millasml) [reply src/types/Journal.ts:3]: do this for all the types

File: src/types/Journal.ts (hunk 1 above)
Comment (millasml) [reply src/types/Journal.ts:4]: do this for all the types

File: src/types/Journal.ts (hunk 2 above)
Comment (millasml) [reply src/types/Journal.ts:5]: do this for all the types

**Review by millasml — CHANGES_REQUESTED** [review]
So close!!

File: src/types/Journal.ts (hunk 1 above)
Comment (ydymegan) [reply src/types/Journal.ts]: updated!

File: src/types/Journal.ts (hunk 2 above)
Comment (ydymegan) [reply src/types/Journal.ts]: updated!

File: src/types/Journal.ts (hunk 3 above)
Comment (ydymegan) [reply src/types/Journal.ts]: updated!

**Review by millasml — APPROVED** [review]
LGTM!!

**Pull request merged**